frame/utility/src/lib.rs

```diff
@@ -609,7 +609,7 @@ impl<T: Trait> Module<T> {
 	///
 	/// NOTE: `who` must be sorted. If it is not, then you'll get the wrong answer.
 	pub fn multi_account_id(who: &[T::AccountId], threshold: u16) -> T::AccountId {
-		let entropy = (b"modlpy/utilisuba", who, threshold).using_encoded(blake2_256);
+		let entropy = (b"modlpy/multisuba", who, threshold).using_encoded(blake2_256);
```

Comment:
This breaks existing functionality. Is this a typo? It's the same prefix that is used in `sub_account_id` in line 602.

Reply:
if there are any multisigs deployed on Kusama, changing this value would make them completely inaccessible, so I would personally revert this change even if it is a typo.

Reply:
Removed this breaking change from the PR in 3f2b74370c393fabe1d2f96619d5b998eb3df2f6